Automattic/wp-calypso
# Analytics: Cleanup unused `statsd` code

#### Changes proposed in this Pull Request

Since we removed perfmon in #54748, some `statsd` and timing analytics functions became obsolete. This PR cleans them up.

#### Testing instructions

* Verify Calypso builds well.
* Verify analytics tests pass: `yarn run test-client analytics`
